### PR TITLE
Fixed link errors (part of issue #208)

### PR DESCRIPTION
**Pull request by SebastianZ**

### CODE REVIEW

**Comment by frivoal** [comment]
The multicol part of this patch looks good. For other specs, see with their respective editors.

**Comment by SebastianZ** [comment]
Thank you for the review, Florian! Should I do something to get a review from the other spec. authors?

Sebastian

File: css-logical-props/Overview.bs [hunk 1]
```diff
@@ -22,7 +22,7 @@ Introduction</h1>
   This mapping controls the interpretation of logical keywords and properties.
 
 <h2 id="logical-controls">
-Logical Directional Values: ''block-start'', ''block-end'', ''inline-start'', ''inline-end''</h2>
+Logical Directional Values: 'block-start', 'block-end', 'inline-start', 'inline-end'</h2>
```
Comment (tabatkins) [inline css-logical-props/Overview.bs]: These (and the next) are wrong - `'foo'` is the shorthand for property autolinks, `''foo''` is for values, and is being used correctly.

File: css-logical-props/Overview.bs [hunk 2]
```diff
@@ -10,7 +10,7 @@ ED: https://drafts.csswg.org/css-logical-props/
 Editor: Rossen Atanassov, Microsoft, ratan@microsoft.com
 Editor: Elika J. Etemad / fantasai, Invited Expert, http://fantasai.inkedblade.net/contact
 Abstract: This module introduces logical properties and values that provide the author with the ability to control layout through logical, rather than physical, direction and dimension mappings. The module defines logical properties and values for the features defined in [CSS21]. These properties are writing-mode relative equivalents of their corresponding physical properties.
-Ignored Terms: div, dl, dfn
+Ignored Terms: div, dl, dfn, block-start, block-end, inline-start, inline-end, start, end
```
Comment (tabatkins) [inline css-logical-props/Overview.bs]: These shouldn't be ignored; they're terms that need to link up in this spec. If the definitions are broken that's something to be fixed, not ignored.

**Comment by tabatkins** [comment]
I reviewed all of them. Left some comments; the rest are good.

File: css-logical-props/Overview.bs (hunk 2 above)
Comment (SebastianZ) [inline css-logical-props/Overview.bs]: Ok, I've addressed this and the next comment in https://github.com/w3c/csswg-drafts/pull/212/commits/4447eecd147b717e522e717e7aa9dfe7ddac76d6.

> If the definitions are broken that's something to be fixed, not ignored.

My thought was that they are already defined in the prose. Those values affect many different properties in several specs, so I wonder how that can be defined correctly.

Sebastian

**Pull request merged**